### PR TITLE
Add more crafting recipes for storage items

### PR DESCRIPTION
**Pull request by AtomicFox556**
#### Summary
Content "Add more crafting recipes for storage items"

#### Purpose of change
Improve the variety of craftable gear that can be used to store items, allowing players to fine-tune their gear if necessary.

#### Describe the solution
Add recipes for various belts, pouches, bags and other things which are worn on belted layer; recipes are based on previously-existing recipes of similar items. Also add a magnet item that is necessary for crafting a variant of load-bearing vest; someone could later add it to other recipes as well.

#### Describe alternatives you've considered
Certain items for which the recipes were added include steel as one of the materials from which they're made, presumably for minor metal parts of it. I didn't make it a requirement to use blacksmithing tools to make those items, but could add it if it proves necessary.

#### Testing
Launched the game, made sure that no errors happen.

#### Additional context

None.

### CODE REVIEW

File: data/json/recipes/tools/containers.json [hunk 1]
```diff
@@ -507,6 +507,19 @@
...
+    "result": "storage_line",
+    "type": "recipe",
+    "activity_level": "LIGHT_EXERCISE",
+    "category": "CC_OTHER",
+    "subcategory": "CSC_OTHER_CONTAINERS",
+    "skill_used": "tailor",
+    "difficulty": 1,
+    "time": "1 h",
+    "autolearn": true,
+    "using": [ [ "tailoring_nylon_patchwork", 2 ] ],
```
Comment (TheShadowFerret) [inline data/json/recipes/tools/containers.json:520]: Where did the carabiners on this go 🤔

Comment (AtomicFox556) [reply data/json/recipes/tools/containers.json:520]: Fixed, recipe now needs some metal as well.

**Review by bombasticSlacks — APPROVED** [review]
I'm not much of a crafting connoisseur but these look good to me

**Review by Maleclypse — CHANGES_REQUESTED** [review]
Please add "alnico" to the dictionary file so that it doesn't break all PRs merged after this on the spell checker.

**Comment by AtomicFox556** [comment]
> Please add "alnico" to the dictionary file so that it doesn't break all PRs merged after this on the spell checker.

Done.

**Pull request merged**